tests/unit/test_readers.py

```diff
@@ -34,45 +18,25 @@ def test_read_invalid_extenstion() -> None:
 def test_gt3x_loader() -> None:
     """Test the gt3x loader."""
     test_file = pathlib.Path(__file__).parent / "sample_data" / "example_actigraph.gt3x"
-    watch_data = readers.gt3x_loader(test_file)
+    watch_data = readers.read_watch_data(test_file)
     assert isinstance(watch_data, WatchData)
     assert isinstance(watch_data.acceleration, Measurement)
     assert isinstance(watch_data.lux, Measurement)
     assert watch_data.temperature is None
```

Comment:
If we're doing these asserts is there a reason not to check capacitance and battery for the gt3x files ?

Reply:
No, not really besides laziness. Since we reduced the numbers of tests I'll add a check for all possible measurements. I guess in theory there's a chance for a short enough file that there wouldn't be all the sensors (we are talking order of one minute, but then we probably won't analyze those files anyway).